config_utilities_ros/config_utilities_ros/gui/README.md

```diff
@@ -0,0 +1,37 @@
+# Config Utilities Dynamic Config GUI
+
+# Installation
+
+## Dependencies
+Local:
+```bash
+pip install Flask webbrowser
+```
+
+System:
+```
+sudo apt install python3-flask python3-webbrowse
```

Comment:
(minor) You can add `flask` as a rosdep (`python-flask`) and `webbrowser` is a standard library module that doesn't have to be installed as far as I can tell?